iscservice/demo/main.c

```diff
@@ -0,0 +1,578 @@
+/* -*- C -*- */
```

Comment:
Please rename the file to "m0iscdemo.c".

Reply:
Ok, done.